src/test/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodecRegistryTest.java

```diff
@@ -337,6 +343,32 @@ private static Stream<Arguments> validCodecToCQLTestCasesCollections() {
             Set.of(-0.75, 42.5)));
   }
 
+  private static Stream<Arguments> validCodecToCQLTestCasesMaps() {
```

Comment:
Can you create an issue to write tests for codec for CQL->Json? We need to mock rock and test the behavior for non String key maps?

Reply:
Thinking loud this needs to be done for all Operations of api tables to check behavior of unsupported datatypes.

Reply:
I am not sure we really need exhaustive testing, as long as some are spot checked on per main CQL type basis (rest should follow).

But I will add a failing case for CQL->JSON for unsupported key type.